sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminClient.java

```diff
@@ -0,0 +1,350 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.personalizer;
+
+import com.azure.ai.personalizer.models.PersonalizerEvaluation;
+import com.azure.ai.personalizer.models.PersonalizerEvaluationOptions;
+import com.azure.ai.personalizer.models.PersonalizerLogProperties;
+import com.azure.ai.personalizer.models.PersonalizerModelProperties;
+import com.azure.ai.personalizer.models.PersonalizerPolicy;
+import com.azure.ai.personalizer.models.PersonalizerPolicyReferenceOptions;
+import com.azure.ai.personalizer.models.PersonalizerServiceProperties;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Context;
+
+import java.util.List;
+
+/**
+ * Client to perform administrative operations on Personalizer instance in a synchronous manner.
+ */
+public final class PersonalizerAdminClient {
```

Comment:
Missing Admin client Builder?

Reply:
I needed only one clientbuilder since I had both the AdminClient and the Client in the same package.